packages/chain-bridge/src/adapters/LayerZeroAdapter.sol

```diff
@@ -273,6 +490,47 @@ contract LayerZeroAdapter is
                         INTERNAL FUNCTIONS
     //////////////////////////////////////////////////////////////*/
 
+    /**
+     * @notice Validates asset and amount, returns OFT contract
+     * @param asset The ERC20 token address
+     * @param amount The amount to transfer
+     * @return oft The OFT contract address for this asset
+     */
+    function _getOFTForAsset(
```

Comment:
Opinion: I understand that it is convenient to revert in this function, but I find the `getXXX` nomenclature a bit misleading. Could we use `_resolveOFTForAsset` instead? For me the resolve keyword indicates a more complex process that could fail. Just my 2 cents